Documentation/HowToGuides/AddingAnInteractor/README.md

```diff
@@ -0,0 +1,68 @@
+# Adding An Interactor
+
+> * Level: Beginner
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The first step into interacting with virtual objects is to define an interactor, which is a concept of something that knows when it is touching an interactable object as well as being able to grab the object to affect it in some way.
+
+The Interactor prefab can be added to any GameObject that we want to be able to touch and grab interactable objects. In this guide we’ll look at making the virtual controllers work as interactors so we can simulate picking up virtual objects with our virtual hands.
+
+## Prerequisites
+
+* [Install the Tilia.CameraRigs.TrackedAlias.Unity] package dependency in to your [Unity] project.
+* [Install the Tilia.Interactions.Interactables.Unity] package dependency in to your [Unity] project.
+
+## Let's Start
+
+### Step 1
+
+The `Interactions.Interactor` prefab comes with an `ExampleAvatar` GameObject, which provides a basic representation of the Interactor containing a simple cube mesh and a simple cube collider. These can be changed to suit whatever purpose you may have, but for this example, we’ll keep the `ExampleAvatar` GameObject untouched.
+
+> If you followed the [Adding A TrackedAlias] guide and added in custom GameObjects for controller avatars, you can delete them now as the `ExampleAvatar` GameObject will provide us with a basic controller avatar instead.
```

Comment:
the reference to `[Adding A TrackedrAlias]` needs updating to the actual link used in the prerequisites, which has now been updated to:

```
[Add the Tilia.CameraRigs.TrackedAlias.Unity]
```